x-pack/filebeat/module/ibmmq/errorlog/_meta/fields.yml

```diff
@@ -0,0 +1,42 @@
+- name: errorlog
+  description: IBM MQ error logs
+  example: Please add example
+  type: group
+  fields:
+  - name: installation
+    description: Please add description
```

Comment:
@felix-lessoer I need some details here on the description.

Reply:
This is the installation name which can be given  at installation time. Default: Installation 1

From IBM documentation

Each installation of IBM® MQ on UNIX, Linux, and Windows, has a unique identifier known as an installation name. The installation name is used to associate things such as queue managers and configuration files with an installation.

Reply:
updated. I wonder if we should in some cases even provide links to the docs.